packages/markdown-medic/src/check.ts

```
@@ -0,0 +1,77 @@
+  rule: string;
+  message: string;
+  line?: number;
+  column?: number;
+  severity: 'error' | 'warning';
+}
+
+export async function checkMarkdown(
+  content: string,
+  options: CheckOptions
+): Promise<CheckResult[]> {
+  const results: CheckResult[] = [];
```

ESLint

[error] Array type using 'CheckResult[]' is forbidden. Use 'Array<CheckResult>' instead.

(@typescript-eslint/array-type)